Dockerfile

```diff
@@ -89,19 +95,13 @@ RUN wget https://github.com/cloudposse/sudosh/releases/download/${SUDOSH_VERSION
     chmod 755 /usr/bin/sudosh
 
 ## Install Duo
-COPY --from=duo-builder dist dist
-RUN make --directory=dist install && \
-    rm -rf dist
+COPY --from=duo-builder dist /
 
 ## Install Google Authenticator PAM module
-COPY --from=google-authenticator-libpam-builder dist dist
-RUN make --directory=dist install && \
-    rm -rf dist
+COPY --from=google-authenticator-libpam-builder dist /
 
 ## Install OpenSSH Portable
-COPY --from=openssh-portable-builder dist dist
-RUN make --directory=dist install && \
-    rm -rf dist
+COPY --from=openssh-portable-builder dist /
```

Comment:
use `dist/ /` for symmetry (also a habbit of using `rsync` - you almost always want to match slashes)